db/migrate/20231206114044_remove_created_at_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCreatedAtFromUsers < ActiveRecord::Migration[7.1]
+  def change
+    remove_column :users, :createdAt    
+  end
+end
```

Comment:
- [ ] Your app seems to not start because of this particular migration file.  The specific problem is related to a column named "createdAt" in the "users" table, which seems to be referenced in the migration file 20231206114044_remove_created_at_from_users.rb. However, it appears that the column doesn't actually exist in the "users" table.

### View:
![image](https://github.com/AbuTalha3/my_blog_rails/assets/50990925/1ce65dca-283e-4bee-baf4-3add56d996a4)